actions/maven/publish/action.yml

```diff
@@ -86,7 +90,10 @@ runs:
         GPG_KEY_PASS: "${{ inputs.gpg-key-pass }}"
         SLSA_DIR: "${{ inputs.provenance-download-name }}"
         PROVENANCE_FILES: "${{ inputs.provenance-download-name }}"
+        TARGET_DIR: "${{ inputs.target-download-name }}"
       run: |
+        # Change name of downloaded target directory
+        mv "${TARGET_DIR}" ./target
```

Comment:
did you verify this change is needed? I think the secure-download Action will download artifact in a folder that has the name of the original folder uploaded, not the randomized name.